scripts/build_distro.sh

```diff
@@ -18,6 +18,17 @@ distro=$3
     cd ${topdir}/build/
 
     ROOTFS_DIR=${topdir}/build/${build}/tisdk-${distro}-${machine}-rootfs
+    enable_weston ${ROOTFS_DIR}
+}
+
+function enable_weston() {
```

Comment:
Instead of enable_weston, Can you make it a bit more generic.

Like `setup_target_configs()` or something like that so that this function can be used not just for weston but for copying any other configs in future.

Reply:
Also, calling the function is fine but the implementation of the function should be machine specifics. Use if or switch case whichever makes sense.

Because these weston configs are needed only for AM62x and not AM64x.